package.json

```
@@ -3,10 +3,12 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "generate": "true",
-    "format": "prettier --write --ignore-path .gitignore ."
+    "generate": "./scripts/generate.sh && npm run format",
+    "format": "poetry run black ."
   },
   "devDependencies": {
+    "@seamapi/nextlove-sdk-generator": "^1.7.7",
+    "@seamapi/types": "^1.149.0",
```

Let pin the version numbers in this case
```suggestion
    "@seamapi/nextlove-sdk-generator": "1.7.7",
    "@seamapi/types": "1.149.0",
```

---

https://github.com/seamapi/python-next/pull/6/commits/90a74fec3905cacb0cac698aa60506df679e9487